### PR TITLE
Adding other services block

### PR DESCRIPTION
**Pull request by votemike**
Waiting on reply to JIRA comment https://jira.dev.bbc.co.uk/browse/PROGRAMMES-5512?focusedCommentId=3417246&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3417246

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
Can't figure out how to delete comments. But no longer waiting for any responses to ticket questions...

File: app/Resources/views/schedules/by_day.html.twig [hunk 1]
```diff
@@ -24,6 +24,26 @@
         {% endfor %}
     </ol>
 
+    {# If there is only 1 service in the network, it is the service we are currently viewing #}
+    {% if servicesInNetwork|length >= 2 %}
+        {% if service.network.medium == constant('BBC\\ProgrammesPagesService\\Domain\\Enumeration\\NetworkMediumEnum::TV') %}
```
Comment (BPScott) [inline app/Resources/views/schedules/by_day.html.twig]: Would putting this check in the Controller be nicer? Exposing 'isTvNetwork' passed into the view perhaps? The view layer feels like the wrong layer of abstraction to be checking PHP constants.

Comment (votemike) [inline app/Resources/views/schedules/by_day.html.twig]: Yeah, I guess that would be better.

**Comment by votemike** [comment]
R.E. the CSS. Since the programmes stuff wasn't styled, I assumed this didn't need to be styled yet.

**Comment by BPScott** [comment]
Added som small niggles. This is :+1: now.

**Pull request merged**